README.md

```diff
@@ -1,33 +1,94 @@
+  </a> -->
+
+  <h1 align="center">GYFT</h1>
+
+  <p align="center">
+    <i>Get Your Freaking Timetable</i>
+    <br />
+    <a href="https://github.com/metakgp/gyft/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/metakgp/gyft/issues">Request Feature</a>
+  </p>
+</div>
+
+
+<!-- TABLE OF CONTENTS -->
+<details>
+<summary>Table of Contents</summary>
+
+- [About The Project](#about-the-project)
+- [Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+- [Usage](#usage)
+- [Development Notes](#development-notes)
+- [Contributing](#contributing)
+- [Maintainer(s)](#maintainers)
+
+</details>
+
+
+<!-- ABOUT THE PROJECT -->
+## About The Project
+
+Gets your timetable from ERP and adds it to your Google Calendar or gives you an ICS file which you can add in any common calendar application.
 
 > **Note** All updates to this repo should reflect, with appropriate refactorisation, in [gyft-serve](https://github.com/metakgp/gyft-serve/)
 
+<p align="right">(<a href="#top">back to top</a>)</p>
 
-## Updates To Be Made Each Sem
+<!-- GETTING STARTED -->
+## Getting Started
 
-The following procedure is to be followed each new semester by the maintainer for updating GYFT to work properly.
+To set up a local instance of the application, follow the steps below.
 
-- Download the academic calendar for the particular year.
-- New semester dates need to be updated in dates.py
-- The format for the date-time is `(YYYY, MM, DD, HH, MM)`.
-- Raise a pull request once the dates are updated.
+### Prerequisites
+The following prerequisites are required to be installed for the project to function properly:
+* Python 3
+* IIT-KGP Student [ERP](https://erp.iitkgp.ac.in) Account
+
+<p align="right">(<a href="#top">back to top</a>)</p>
 
-## How to use the program?
-
-1. Clone this repository 
-2. Install the dependencies by running:
-    ```sh
-    pip install -r requirements.txt
-    ```
-3. Run the gyft script:
-    ```sh
-    python3 gyft.py
-    ```
+### Installation
+
+_Now that the environment has been set up and configured to properly compile and run the project, the next step is to install and configure the project locally on your system._
+1. Clone the repository
+   ```sh
+   git clone --depth 1 https://github.com/metakgp/gyft
+   ```
+2. Change current directory to the project directory
+   ```sh
+   cd gyft
+   ```
+3. Install the dependencies
+   ```sh
+   pip install requirements.txt
+   ```
+4. Run the script
+   ```sh
+   python3 gyft.py
+   ```
     - Optional flags: 
     	- `--output` or `o`: Output file (`.ics`) to which the ICS file will be written. **Default:** `timetable.ics`
-    	- `--del-events`: To delete events automatically added by the script before adding new events (explained later) **Default:** `False`
+    	- `--del-events`: To delete events automatically added by the script before adding new events (explained later) **Default:** `False` 
```

Comment:
Shift this part to `Usage` section. 
This is only intended to get the things setup for gyft.

Add all the details which aren't there in `Usage` section from here and format it.

Reply:
So like all this; points 4 to 6? 
To `usage` section?
![image](https://github.com/metakgp/gyft/assets/139053348/1d46e0f3-31b8-49e8-bda7-22c6032bc732)

Reply:
yes, this is how you "use" not install/set it up

Reply:
Perfect!
And it's fine to commit, there is nothing bad in getting more suggestions.